app/javascript/application.js

```diff
@@ -19,11 +19,11 @@ import "bootstrap";
 require("@rails/ujs").start();
```

Comment:
Merging all packs into one means we are shipping more javascript then strictly necessary for different parts of the application.
I see why we would want this change (it is easier to manage, packs are cached across applications parts), but can you verify how much bigger the pack gets?

Reply:
The default directory for webpacker (src) packs was changed from app/javascript/packs to app/javascript. The PR reflects this. I don't think it changes the way the packs get sent to the clients though. Webpacker still seems to give multiple output js files.

Reply:
Ah you're right, my bad